recorder.go

```diff
@@ -355,7 +420,8 @@ func (r *Recorder) Flush() {
 	r.reportInFlight = true
 	r.lock.Unlock() // unlock before making the RPC itself
 
-	resp, err := r.backend.Report(r.auth, req)
+	// Question: Where does context come in?
```

Comment:
context.Background() is appropriate here, since we're a background thread and there's no user activity tied to this Flush.